docs/wiki/deployment/file-integrity-monitoring.md

```diff
@@ -145,3 +153,36 @@ File accesses on Linux using `inotify` may incur unexpected and unwanted perform
 The above configuration snippet will enable file integrity monitoring for `homes`, `etc`, and `tmp` but only enable access monitoring for the `homes` and `etc` directories.
 
 > NOTICE: The hashes of files will not be calculated, to avoid generating additional access events.
+
+## Troubleshooting FIM
+
+Sometimes, despite a correct osquery configuration, the file events tables don't receive any events.
+
+First, make sure you are launching osquery as root/Administrator.
+
+In some cases, the problem might be interference from additional security permissions settings:
+
+- On CentOS-like systems, check that the SElinux settings are not preventing osquery from performing FIM.
+- On Debian-like systems, check that AppArmor is not blocking osquery.
+- On macOS, the `osqueryd` agent (or `Terminal.app`, if using `osqueryi`) may need Full Disk Access permissions, in Security and Privacy settings.
+
+Also, remember you can run `osqueryd` with the `--verbose` flag to see if any helpful warnings appear.
+
+If you're attempting to use FIM on Linux via Audit and you see an error like the following, your system is probably configured to use `auditd`; unfortunately osquery cannot share access to the Audit subsystem with `auditd`. You can use osquery with one of the other FIM sources, or discontinue the use of `auditd`.
+
+```text
+osquery> I1107 17:49:42.229321  8235 auditdnetlink.cpp:601] Failed to set the netlink owner
+```
+
+Last but not least, see the troubleshooting guidance in [process auditing with osquery](./process-auditing.md), which similarly covers the topic of event-based tables in osquery and its use of the Audit subsystem on Linux.
+
+## Known Issues
+
+Implementing FIM across all platforms and using multiple sources means that there are a few problematic corner cases. This is not an exhaustive list, but rather, some long-standing issues to be aware of. If you encounter one that is not in our tracked issues, please submit it.
+
+- On some platforms, it may not be possible to monitor a given path, until a file or directory already exists at that path. [Issue 3212](https://github.com/osquery/osquery/issues/3212)
+- If a watched file is deleted, iNotify stops watching that path. [Issue 6495](https://github.com/osquery/osquery/issues/6495)
+- With iNotify, moving a directory of directories into a watched directory does not immediately add all of those subdirectories to the watched set. [Issue 1969](https://github.com/osquery/osquery/issues/1969)
```

Comment:
Heads up about the inconsistent spelling of inotify, we use "inotify", "`inotify`", and "iNotify". I am unsure which is correct, but I recommend we rewrite the document to use a single forme.